packages/dev-frontend/src/components/Dashboard/Chart/context/ChartProvider.tsx

```diff
@@ -120,34 +114,50 @@ async function fetchData(API_URL: string, query: string) {
 };
 
 export const ChartProvider: React.FC<FunctionalPanelProps> = ({ children, loader })  => {
-  const timestamps: Array<number> = createListOfTimestamps();
-  const [tvl, setTvl] = useState<Array<tvlData>>();
-  const [isMounted, setIsMounted] = useState<boolean>(true);
-
+  const timestamps: Array<TimestampsObject> = createListOfTimestamps();
+  const [ isTVLDataAvailable , SetisTVLDataAvailable ] = useState<boolean>(true);
```

Comment:
should this be setIsTVLDataAvailable

Reply:
https://github.com/Threshold-USD/dev/pull/56/commits/3c001e46a13991a21d49412d1d0c9fb0d027eed2